src/deluge/playback/playback_handler.cpp

```diff
@@ -2489,6 +2489,11 @@ bool PlaybackHandler::tryGlobalMIDICommands(MIDIDevice* device, int channel, int
 				}
 				break;
 
+			case GlobalMIDICommand::FILL:
+				currentSong->fillModeActive = true;
+				indicator_leds::setLedState(IndicatorLED::SYNC_SCALING, true);
```

Comment:
Why do you light up the SYNC_SCALING led? What if the user didn't enable to use the sync scaling button for fills, and instead uses it for the default behaviour or a different community feature?

Reply:
Yeah, the sync scaling code is leftover from the initial plan to take over the sync scaling button. Next step after global MIDI command is to add a community setting to choose which physical button triggers fill mode. Or none, of course, if the user just wants to use it via MIDI.